diff --git a/src/napari_ism/_widget.py b/src/napari_ism/_widget.py
index 92b03d1..01b0c80 100644
--- a/src/napari_ism/_widget.py
+++ b/src/napari_ism/_widget.py
@@ -14,7 +14,7 @@ from brighteyes_ism.analysis.Tools_lib import fingerprint
 import brighteyes_ism.simulation.PSF_sim as ism
 
 from matplotlib.backends.backend_qt5agg import FigureCanvas
-from PyQt5 import QtWidgets
+from qtpy import QtWidgets
 
 from magicgui import magicgui
 
